.travis.yml

```
@@ -7,8 +7,8 @@ cache:
     vendor
 
 php:
-  - 5.6
-  - 7.2
+  - 5.6.37
+  - 7.2.6
```

Using `7.2` works, so let's keep it at that version for now.
```suggestion
  - 7.2
```